app/shared/compact-reservation-list/CompactReservationList.js

```diff
@@ -18,8 +19,16 @@ class CompactReservationList extends Component {
       <li key={reservation.begin}>
         {resourceName ? `${resourceName}: ` : ''}
         <TimeRange begin={reservation.begin} end={reservation.end} />
+        {removable &&
+          <Glyphicon
+            glyph="remove-circle"
+            onClick={() => this.props.onRemoveClick(reservation.begin)}
```

Comment:
I can't get the reservation from the props, so even if there would be a handleRemoveClick function, i would need to invoke it from the component like {() => this.handleRemoveClick(reservation)}